src/api/apiHandler.js

```diff
@@ -12,6 +17,13 @@ export const apiHandler = async (endpoint, method = 'GET', requestBody) => {
         `http://localhost:3001${endpoint}`,
         requestOptions
     );
-
     return response;
 };
+
+export const authorisedApiHandler = (endpoint, method = 'GET', requestBody) => {
+    const token = localStorage.getItem('authToken');
```

Comment:
usually they called 'accessToken' and 'refreshToken'

Reply:
done

Reply:
i can't see a PR for that